frontend/src/concepts/connectionTypes/ConnectionTypeDetailsHelperText.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {
+  Button,
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  HelperText,
+  HelperTextItem,
+  LabelGroup,
+  Popover,
+} from '@patternfly/react-core';
+import { getDescriptionFromK8sResource } from '~/concepts/k8s/utils';
+import TruncatedText from '~/components/TruncatedText';
+import { ConnectionTypeConfigMapObj } from './types';
+import UnspecifiedValue from './fields/UnspecifiedValue';
+import CategoryLabel from './CategoryLabel';
+
+type Props = {
+  connectionType?: ConnectionTypeConfigMapObj;
+};
+
+export const ConnectionTypeDetailsHelperText: React.FC<Props> = ({ connectionType }) => {
+  const displayName =
+    connectionType && connectionType.metadata.annotations?.['openshift.io/display-name'];
```

Comment:
I think if this is not a preview, we want to fall back to the metadata name therefore using the util getDisplayNameFromK8Resource